README.md

```diff
@@ -50,6 +50,27 @@ It is available for download while running the service at `http://localhost:port
 gets generated from code (specifically from Spring annotations) 
 
 ## Authentication Providers
+### Enabling and Selecting Authentication Providers
+The Login Service allows users to select which authentication providers they would like to use
+as well as the order in which the authentication methods get prioritized.
+
+This is done by setting a number (Greater than 0) on the "enable" tag for each auth method.
+the number used indicates the order in which you would like to use the method.
+
+For Example:
+The 2 methods currently enabled are config specified users:
+```
+logingw.rest.users.enable = 1
```

Comment:
Can `users` config be prefixed with `auth.` as in LDAP case?

Reply:
Ah, yeah, makes sense to me to put it all under one tag. Will do